include/albatross/src/core/priors.hpp

```diff
@@ -15,6 +15,7 @@
 
 namespace albatross {
 
+constexpr double LOG_2_ = 0.6931471805599453;
```

Comment:
Is it obvious to you these are the natural base? I'm quite used to LN, or base 2

Reply:
I feel pretty ok about it considering it's the result of `log(2)` (and `log(2 * pi)` below this line).  If you have strong opinions though I could modify the name somehow.